apps/frontend/src/components/MainNavigation/MainNav.tsx

```diff
@@ -67,6 +92,12 @@ const MainNav = () => {
               <Nav.Link href="#pricing" className={styles.navbarCollapseLink}>
                 NOTIFICATIONS
               </Nav.Link>
+              <Nav.Link className={styles.navbarCollapseLink} onClick={() => {
+                  deleteStoredUser()
+                  window.location.reload()
+              }}>
+                LOG OUT
```

Comment:
I think "Sign out" is more common for this

Reply:
Done!